CMakeLists.txt

```diff
@@ -413,6 +413,9 @@ else ()
   set(MLPACK_INCLUDE_DIRS ${MLPACK_INCLUDE_DIRS} "${ENSMALLEN_INCLUDE_DIR}")
 endif ()
 
+find_package(cereal "${CEREAL_VERSION}" REQUIRED)
+set(MLPACK_INCLUDE_DIRS ${CEREAL_INCLUDE_DIR})
```

Comment:
This doesn't preserve the existing directories that are already in `MLPACK_INCLUDE_DIRS`.

Reply:
True, you are right, a small mistake, sorry